src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -788,18 +787,15 @@ private MenuBar createMenu() {
 
                 new SeparatorMenuItem(),
 
-                factory.createMenuItem(StandardActions.SEND_AS_EMAIL, new OldDatabaseCommandWrapper(Actions.SEND_AS_EMAIL, this, Globals.stateManager)),
-
-                new SeparatorMenuItem()
-
+                factory.createMenuItem(StandardActions.MANAGE_KEYWORDS, new ManageKeywordsAction(this))
         );
 
         if (Globals.prefs.getBoolean(JabRefPreferences.SPECIALFIELDSENABLED)) {
             boolean menuItemAdded = false;
-            if (Globals.prefs.getBoolean(JabRefPreferences.SHOWCOLUMN_RANKING)) {
+            /*if (Globals.prefs.getBoolean(JabRefPreferences.SHOWCOLUMN_RANKING)) {
```

Comment:
Why is this commented out?

Reply:
Two people said in #4669 we will remove that. I have preferred to leave it as a comment, waiting for more opinions, but I have forgotten the `[WIP]` prefix (this is my first contribution to this project ;-)).

Do I put it back?

Reply:
I have put it back for the moment.

Thanks for your review!

Reply:
I took care of this in https://github.com/JabRef/jabref/pull/4708 ;-).